test/erlfmt_format_SUITE.erl

```diff
@@ -232,6 +238,30 @@ literals(Config) when is_list(Config) ->
     ?assertSame("_Bar\n"),
     ?assertFormat("$ ", "$\\s\n").
 
+-if(?OTP_RELEASE >= 27).
```

Comment:
I don't think this needs to be a compile-time check. A runtime check would be enough since it doesn't use any of the new features inline.

I would suggest adding a `otp_27_features` group and then have an `init_per_group` callback with the check:
```erl
init_per_group(otp_27_features, Config) ->
    case erlang:system_info(otp_release) >= "27" of
        true -> Config;
        false -> {skip, "Skipping tests for features from OTP >= 27"}
    end;
init_per_group(_, Config) ->
    Config.
```